packages/web-frontend/src/actions.js

```diff
@@ -541,10 +541,10 @@ export function setMeasure(measureId) {
  *
  * @param {object} measureConfig
  */
-export function updateMeasureConfig(measureId, measureConfig) {
+export function updateMeasureConfig(mapOverlayIds, measureConfig) {
```

Comment:
```suggestion
export function updateMeasureConfig(mapOverlayId, measureConfig) {
```

Reply:
Will leave that for now because we might need to update configs for both selected map overlays in PHX-1?

Reply:
The code is used with a single id: https://github.com/beyondessential/tupaia/blob/dev/packages/web-frontend/src/sagas.js#L934

Let's use `mapOverlayId` so that we are consistent with the current code, and refactor it if indeed we will need to change it to receive an array of ids

Reply:
The state may look a bit different when multiple overlays can be selected, and it's a bit hard to predict from now, so it's safer to be consistent with the existing code and refactor at a later stage

Reply:
Sounds good, let's keep it consistent.